authors/apps/articles/serializers.py

```diff
@@ -26,4 +26,13 @@ class Meta:
             'image',
             'average_rating'
         ]
-        
\ No newline at end of file
+
+class FavoriteAnArticleSerializer(serializers.ModelSerializer):
+
+    article = ArticleSerializer(required=False)
+    favorited_by = UserSerializer(required=False)
+
+    class Meta:
+        fields = '__all__'
```

Comment:
@ alex good work but please help me understand why you would use `__all__` on line 36

Reply:
@jennizalwango using `__all__` to indicate that all fields in the model should be used.